hicexplorer/hicBuildMatrix.py

```diff
@@ -136,6 +136,12 @@ def parse_arguments(args=None):
                         type=argparse.FileType('w'),
                         required=True)
 
+    parser.add_argument('--outFileNameLog',
+                        help='Output file name for the log file. If not given, a file with the same name as'
+                             'the matrix, ending in .log will be saved',
+                        metavar='FILENAME',
+                        type=argparse.FileType('w'))
+
```

Comment:
@bgruening heads up, now hicBuildMatrix will save a log file. The file name can be given as parameter or if not set, the same name of the output matrix will be used but with .log This change was necessary to make the new hicQC program to work.